test/test_sid/Makefile

```diff
@@ -1,3 +1,5 @@
+PYANG?=../../bin/pyang
```

Comment:
PYANG is supposed to be set when the tests are run

Reply:
How does it get set if you don't source env.sh?